ai-service/src/ai_service/main.py

```
@@ -1,53 +1,91 @@
+    try:
+        print("Scanning project directory ...")
+        code_files = project_ingestor.scan_code_files(project_dir)
+        print(f"Found {len(code_files)} code files to process.")
+
+        code_snippets = []
+        embeddings = []
+
+        print("Embedding content for each file ...")
+        for file_path in code_files:
+            try:
+                with open(file_path, "r", encoding="utf-8") as f:
```

Ruff (0.12.2)

Unnecessary mode argument

Remove mode argument

(UP015)